src/frontend/hooks/useGetQuest.ts

```diff
@@ -4,7 +4,7 @@ import { useQuery, useQueryClient } from '@tanstack/react-query'
 export default function useGetQuest(questId: number | null) {
   const queryClient = useQueryClient()
   const queryKey = `getQuest:${questId}`
-  const query = useQuery<Quest>({
+  const query = useQuery<Quest | null>({
```

Comment:
I don't think we need this, react-query already infers the value, no need to use the type

Reply:
without this, I wasn't getting the linting/autocomplete previously. looks like it is working now on my machine without specifying the type though. will make the change 👌